src/mahoji/commands/finish.ts

```diff
@@ -26,16 +26,42 @@ export const finishCommand: OSBMahojiCommand = {
 	],
 	run: async ({ interaction, options }: CommandRunOptions<{ input: string }>) => {
 		await deferInteraction(interaction);
-		const val = finishables.find(i => stringMatches(i.name, options.input));
+		const val = finishables.find(i => stringMatches(i.name, options.input) || i.aliases?.includes(options.input));
```

Comment:
we always check `stringMatches` on aliases, not direct string match. aliases arent included in the first place because theres autocomplete and so not a big need for them, so maybe dont even need to add them.

Reply:
ah right, I fixed the check. A lot of people were requesting aliases to work for this command, I don't think it's bad thing to have them either